README.md

```diff
@@ -95,24 +96,33 @@ If history is enabled, no database type is specified in the `[database]`
 section of the configuration, and no database path has been specified, an
 in-memory database will be used.
 
-## Run
+## Install from source
 
-    cargo run
+### Cargo
 
-## Install
-
-    cargo install
+    cargo install --git https://github.com/nuxeh/url-bot-rs
 
 ### Debian/Ubuntu (linux)
 
+    git clone https://github.com/nuxeh/url-bot-rs
+    cd url-bot-rs
     cargo install cargo-deb
     cargo deb --install
 
-After this, the bot may be started by running `url-bot`
+After this, the bot may be started manually by running `url-bot-rs`.
+
+## Running as a service
+
+### Debian package install
+
+If you install using the Debian package, this also installs a systemd unit for
+running as a service, e.g.:
 
-## Auto-start
+    systemctl enable url-bot-rs.service
+    systemctl start url-bot-rs.service
```

Comment:
This can be shortened to `systemctl enable --now url-bot-rs.service`

Reply:
Or even without `.service`, but that may be unwanted.

Reply:
Nice, i'll update the readme.

Another thought, might it be worth giving instructions for manual install of the unit, e.g.
```
sudo cp systemd/url-bot-rs.service /lib/systemd/system/
sudo chmod 644 /lib/systemd/system/url-bot-rs.service
```
or even:
```
sudo install -m 644 systemd/url-bot-rs.service /lib/systemd/system/
```